test/integration/cloud-config-vmware.ini.template

```diff
@@ -0,0 +1,7 @@
+[DEFAULT]
+vmware_user: @VMWARE_USER
+vmware_password: @VMWARE_PASSWORD
+vmware_host: @VMWARE_HOST
+vmware_https_port: @VMWARE_HTTPS_PORT
```

Comment:
The other @foo variables here match the VMWARE_* environment variables. Could we rename this way `VMWARE_PORT` too?

Reply:
Yeah, I was mainly looking to differentiate between the https port and the ssh port.  Which tests needed ssh access, I can't find references to how that var is being named today?